modules/users/client/services/password-validator.client.service.js

```diff
@@ -11,7 +11,7 @@ angular.module('users').factory('PasswordValidator', ['$window',
         return result;
       },
       getPopoverMsg: function () {
-        var popoverMsg = 'Please enter a passphrase or password with greater than 10 characters, numbers, lowercase, upppercase, and special characters.';
+        var popoverMsg = 'Please enter a passphrase or password with more than 10 characters, numbers, lowercase, uppercase, and special characters.';
```

Comment:
@lirantal @codydaig Why do this when result tells us the errors? I think the popoverMsg should be result.requiredTestErrors if possible.

https://www.npmjs.com/package/owasp-password-strength-test